_includes/home-banner.html

```diff
@@ -6,9 +6,6 @@ <h1 class="hero__title">Origami</h1>
 				<p class="hero__summary">
 					Unifying branding across FT digital products.
 				</p>
-				<p class="hero__summary">
-					Making web development better, faster, stronger.
```

Comment:
But it also mentions unifying the styles down the page, the idea is to point out those two points in the hero, right?